client/src/components/HaloModelForm.vue

```diff
@@ -0,0 +1,135 @@
+<template>
+  <div class="md-layout md-gutter">
+    <div class="md-layout-item">
+      <md-subheader>Halo Model</md-subheader>
+      <md-divider></md-divider>
+      <div class="md-layout md-gutter">
+        <!-- Here is where its repeating to load the input -->
+        <div v-for="(input, inputName) in inputs" :key="input.id" class="md-layout-item">
+          <!--<md-field :class="validationClass">-->
+            <!--<label :for="inputName">{{ input.label }}</label>-->
+            <!--<md-input v-model="input.value" :id="inputName" v-on:input="handleInput"/>-->
+            <InputField
+            :key="inputName"
+            :label="input.label"
+            :step="input.step"
+            :currentValue="input.value"
+            :min="input.min"
+            :max="input.max"
+            :inputName="inputName"
+            :inputType="input.inputType"
+            :options="input.options"
+            :setCurrentValue="createSetCurrentValueFunc(inputName)"
+            />
+            <!--<span class="md-error">Please input a valid number</span>-->
+          <!--</md-field>-->
```

Comment:
Seems like extra comments. Not a big deal, but can't tell if this is needed anymore.